app/src/components/integrations/elements/integrationSettings/integrationSettings.jsx

```diff
@@ -58,9 +58,18 @@ export const IntegrationSettings = (props) => {
   const query = useSelector(querySelector);
   const dispatch = useDispatch();
   const { trackEvent } = useTracking();
-  const availableGlobalIntegrations = globalIntegrations[query.subPage] || [];
-  const availableProjectIntegrations = projectIntegrations[query.subPage] || [];
-  const groupedIntegrations = [...availableGlobalIntegrations, ...availableProjectIntegrations];
+  const availableGlobalIntegrations = useMemo(() => globalIntegrations[query.subPage] || [], [
```

Comment:
Could we combine these 3 useMemo into one?

Reply:
Done.